web/components/layouts/Main/Main.stories.tsx

```diff
@@ -49,6 +49,7 @@ const defaultClientConfig = {
   name: "Spiderman's super serious stream",
   summary: 'Strong Spidey stops supervillains! Streamed Saturdays & Sundays.',
   extraPageContent: 'Spiderman is **cool**',
+  socketPathOverride: '',
```

Comment:
I feel like if we had an actual use case for needing to override the path for the websocket connection that using adding a new property and using for two things could make sense, but in this case it's mostly just being added as a way to disable the websocket. So I'm not sure it's the right answer in this particular case when we're just trying to stop the websocket from connecting in Storybook.

Reply:
Makes sense. I guess we are already using the `storybook-addon-fetch-mock` mock for fetch requests, I can try using that to see if it's able to mock Websocket requests. If not, are you open to using another library to mock Websockets? ( Or if we don't want to burden ourselves with another library, we can also try to mock the `window.Websocket` func )


Regardless, this won't **stop** storybook from making websocket connections, but atleast the connections would be properly handled this time.
What do you think?

Reply:
Good point! That's probably the easiest solution, by wrapping the Storybook story with a component that overwrites the websocket's constructor to just be a no-op. Then it should just silently do nothing. That way it also doesn't change the real code, and only impacts the Story.